libs/components/core/src/lib/modules/shared/number-format/number-format-utility.ts

```diff
@@ -20,9 +20,6 @@ function parseIntAutoRadix(text: string): number {
   return result;
 }
 
-// Need to add the following to classes which contain static methods.
-// See: https://github.com/ng-packagr/ng-packagr/issues/641
-// @dynamic
```

Comment:
Legacy fix for Angular 8 compiler.

Reply:
Did you test that these are no longer needed?

Reply:
I did; the error would have surfaced during a build, but I also tested the bundle in a host SPA to confirm.